site/content/en/_index.md

```diff
@@ -20,6 +20,16 @@ KWOK stands for Kubernetes WithOut Kubelet. So far, it provides two tools:
 - `kwok` is the cornerstone of this project, responsible for simulating the lifecycle of fake nodes, pods, and other Kubernetes API resources.
 - `kwokctl` is a CLI tool designed to streamline the creation and management of clusters, with nodes simulated by `kwok`.
 
+## Why KWOK?
+
+- Lightweight: You can simulate thousands of nodes on your laptop without significant consumption of CPU or memory resources.
```

Comment:
```suggestion
- Lightweight: You can [simulate thousands of nodes] on your laptop without significant consumption of CPU or memory resources.
```
[simulate thousands of nodes]: {{< relref "/docs/user/kwok-manage-nodes-and-pods" >}}